docs/source/starter/installation.rst

```diff
@@ -0,0 +1,63 @@
+.. _installation:
+
+############
+Installation
+############
+
+******************
+Lightning Coverage
+******************
+
+PyTorch Lightning is maintained and tested on Pytorch 3.7.0+ and PyTorch 1.7.0+.
+
+It is rigorously tested across multiple GPUs, TPUs, CPUs and IPUs. GPU tests run on two NVIDIA P100. TPU tests run on Google GKE TPUv2/3.
+TPU py3.7 means we support Colab and Kaggle env. IPU tests run on MK1 IPU boxes.
+
+
+*********************
+Installation with Pip
+*********************
+
+Install any supported version of PyTorch if you want from `PyTorch Installation Page <https://pytorch.org/get-started/locally/#start-locally>`_.
+Now you can install using `pip <https://pypi.org/project/pytorch-lightning/>`_ using the following command:
+
+.. code-block:: bash
+
+    pip install pytorch-lightning
+
+
+***********************
+Installation with Conda
+***********************
+
+If you don't have conda installed, follow the `Conda Installation Guide <https://docs.conda.io/projects/conda/en/latest/user-guide/install>`_.
+Lightning can be installed with `conda <https://anaconda.org/conda-forge/pytorch-lightning>`_ using the following command:
+
+.. code-block:: bash
+
+    conda install pytorch-lightning -c conda-forge
+
+You can also use `Conda Environments <https://docs.conda.io/projects/conda/en/latest/user-guide/tasks/manage-environments.html>`_:
+
+.. code-block:: bash
+
+    conda activate my_env
+    conda install pytorch-lightning -c conda-forge
```

Comment:
What's the benefit of mentioning this? This is not lightning-specific right?

Reply:
I think it's fine. If someone is not familiar with conda at all, then it's just a useful reference.